app/src/componentLibrary/attributeList/editableAttributeList.jsx

```diff
@@ -24,19 +25,27 @@ const NEW_ATTRIBUTE = {
 };
 
 export const EditableAttributeList = ({
-  attributes,
+  value: attributes,
```

Comment:
Why are you decided to rename it?
I think it will be more clear to keep the same naming for the same things here and in `AttributeList`.